src/Forms/SudoModePasswordField.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Forms\FormField;
+
+class SudoModePasswordField extends FormField
+{
+    public const FIELD_NAME = 'SudoModePasswordField';
+    
+    protected $schemaComponent = 'SudoModePasswordField';
+
+    protected $extraClasses = [
+        'SudoModePasswordField',
+    ];
+
+    public function __construct($name, $title = null, $value = '')
+    {
+        // Name must be "SudoModePasswordField" as there's logic elsewhere expecting this
+        // $name and $value are set to null as the react component does not use these arguments
+        parent::__construct(SudoModePasswordField::FIELD_NAME, null, null);
+        // Set title to empty string to avoid rendering a label before the react component has loaded
+        $this->setTitle('');
```

Comment:
That feels like the wrong way to change how a thing renders. I assume the label is being rendered in a template? So we should probably just make sure the template is correct for this new class instead.

Reply:
Trying to use a custom SudoModePasswordField_holder.ss, or using `function FieldHolder() ... return $this->Field()) ends up breaking a bunch of stuff.  It's much simpler and less code to just use the use a single line of code to achieve the same outcome